packages/cms-cli/README.md

```diff
@@ -100,6 +100,31 @@ hs filemanager fetch --portal=DEV [src] [dest]
 hs filemanager fetch --portal=DEV --overwrite [path] [dest]
 ```
 
+Create a new HubDB table
```

Comment:
I think we should consider adding headings for the big subcommands that break into additional subcommands (e.g. `hs hubdb` and `hs filemanager`.

Reply:
I was thinking something like "HubDB Commands" and "File Manager Commands."